src/azul/azulclient.py

```diff
@@ -358,6 +326,38 @@ def wait_for_indexer(self, **kwargs):
         self.queues.wait_for_queue_level(empty=True, **kwargs)
 
 
+def filter_obsolete_bundle_versions(bundle_fqids: Iterable[BundleFQID]) -> List[BundleFQID]:
```

Comment:
Why did you move this method? It makes the diff unnecessarily large so it's hard to detect if you actually modified the method.

Reply:
I was confused by `We need to make it public and remove the leading _ from the name.` I had removed the function from the class entirely.

I amended the changes to the method and have confirmed that the IT works as expected.
Just waiting on Travis now.